README.md

```diff
@@ -4,7 +4,7 @@ This is the repo for the [`xmtp.org`](https://xmtp.org) website, an XMTP communi
 
 This repo is open and contributions from the public are welcome! 🫶
 
-When contributing to `xmtp.org`, please follow the XMTP community [code of conduct]() to help create a safe and positive experience for all.
+When contributing to `xmtp.org`, please follow the XMTP community [code of conduct](https://github.com/osrm/xmtp-dot-org/blob/main/CODE_OF_CONDUCT.md) to help create a safe and positive experience for all.
```

Comment:
Thank you so much for this PR, @osrm! My apologies for the delay in replying - you caught us right in the midst of a doc migration and files were moving around. =)

We keep the XMTP code of conduct as a part of the Community documentation, so it would be great to link folx there instead of creating a new instance of the document. 

So we could link as follows:

```suggestion
When contributing to `xmtp.org`, please follow the XMTP community [code of conduct](https://github.com/xmtp/xmtp-dot-org/blob/main/docs/community/code-of-conduct.md) to help create a safe and positive experience for all.
```

If you could also delete the CODE_OF_CONDUCT.md from this PR, I will approve it for merging!

Thank you again for taking the time to point out the missing link and rectifying it!

Reply:
I've just changed the fix as you suggested. Appreciate your kind suggestion for a better way.